examples/zookeeper/config-and-run.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/bash
 
+# Check if conf folder is empty
+if [ ! "$(ls -A /opt/zookeeper/conf)" ]; then
```

Comment:
The folder is already created via the VOLUME command in the Dockerfile, or the volumeMount in OpenShift. That's why I omitted the folder check. But, it's probably good to check.